app.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
```

Comment:
This is not needed for this end point

Reply:
Do you just mean the express.json part? as i need to use "app"